base/linalg/triangular.jl

```diff
@@ -1087,15 +1174,116 @@ function logm{T<:Union{Float64,Complex{Float64}}}(A0::UpperTriangular{T})
          1.276404810806775e-001,
          2.060962623452836e-001,
          2.879093714241194e-001]
+
```

Comment:
Same as above: what is `theta`, and can we make this routine precision-independent?